registry/storage/driver/s3-aws/s3.go

```diff
@@ -230,6 +243,8 @@ func FromParameters(parameters map[string]interface{}) (*Driver, error) {
 		fmt.Sprint(rootDirectory),
 		storageClass,
 		fmt.Sprint(userAgent),
+		fmt.Sprint(backend),
```

Comment:
@kmala no harm in keeping it as-is, but can you remove it at least from the code you added?